target_chains/solana/examples/send_usd/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "send-usd"
+version = "0.1.0"
+description = "Created with Anchor"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+name = "send_usd"
+
+[features]
+no-entrypoint = []
+no-idl = []
+no-log-ix-name = []
+cpi = ["no-entrypoint"]
+default = []
+
+[dependencies]
+anchor-lang = { workspace = true }
+solana-program = { workspace = true }
+pyth-solana-receiver-sdk = { path = "../../pyth_solana_receiver_sdk"}
```

Comment:
I feel like this should use the public crate since that's how someone would use the sdk